features/specs/gezag-minderjarige/niet-ouder-gezag.feature

```diff
@@ -0,0 +1,218 @@
+#language: nl
+
+Functionaliteit: in verschillende situaties heeft een niet ouder (ook) het gezag over een minderjarige
```

Comment:
ik denk dat we moeten bespreken of we deze feature willen en waarom. Alle regels en scenario's hieronder komen ook al voor in features voor specifieke vragen, alleen is het hier met minder detail en minder context opgenomen.

Wanneer de bedoeling is om te definiëren wat een niet-ouder is, dan denk ik dat je niet echt compleet genoeg en scherp genoeg bent, zie bijvoorbeeld de toelichting op regel 10

Reply:
Besproken tijdens de dagstart. De term _niet ouder_ vervalt en voortaan gebruiken we de term _derde_. De functionele specificatie hiervan wordt meegenomen in de nieuwe specificaties waaraan wordt gewerkt door Cathy, Frank en de afleidingsregelgroep.